README.md

```diff
@@ -1,65 +1,76 @@
-# Bouncer Tests for download.allizom.org
+# Bouncer Tests for http://bouncer-bouncer.stage.mozaws.net
 
-Thank you for checking out Mozilla's [Bouncer](https://wiki.mozilla.org/Bouncer) test suite. Mozilla and [Web QA team](https://quality.mozilla.org/teams/web-qa/) are grateful for the help and hard work of [many contributors](https://github.com/mozilla/bouncer-tests/graphs/contributors) like yourself.
+This repository contains tests for Mozilla's [Bouncer website](https://wiki.mozilla.org/Bouncer).
 
-## Getting involved as a contributor
+[![license](https://img.shields.io/badge/license-MPL%202.0-blue.svg)](https://github.com/mozilla/bouncer-tests/blob/master/LICENSE)
+[![travis](https://img.shields.io/travis/mozilla/bouncer-tests.svg?label=travis)](http://travis-ci.org/mozilla/bouncer-tests/)
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/bouncer.stage.svg?label=stage)](https://webqa-ci.mozilla.com/job/bouncer.stage/)
+[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/bouncer.prod.svg?label=prod)](https://webqa-ci.mozilla.com/job/bouncer.prod/)
+[![requirements](https://img.shields.io/requires/github/mozilla/bouncer-tests.svg)](https://requires.io/github/mozilla/bouncer-tests/requirements/?branch=master)
 
-We love working with contributors to fill out the test coverage for Bouncer Tests, but it does require a few skills. You will need to know some Python and you will need some basic familiarity with [GitHub](https://guides.github.com/).
+## Getting involved
+We love working with contributors to fill out the test coverage for Mozilla's
+support website, but it does require a few skills. By contributing to our test
+suite you will have an opportunity to learn and/or improve your skills with
+Python, Selenium WebDriver, GitHub, virtual environments, the Page Object
```

Comment:
Small nit .. technically bouncer-tests only uses Python, Github, and virtual envs. It's one of our few test suites that doesn't use the Webdriver api.